obspy/signal/rotate.py

```diff
@@ -124,8 +125,8 @@ def rotate_lqt_zne(l, q, t, ba, inc):
         raise ValueError("Back Azimuth should be between 0 and 360 degrees!")
     if inc < 0 or inc > 360:
         raise ValueError("Inclination should be between 0 and 360 degrees!")
-    ba *= 2 * pi / 360
-    inc *= 2 * pi / 360
+    ba = radians(ba)
+    inc = radians(inc)
```

Comment:
Thanks for reverting that part.. also wanted to do that but brain wasn't working anymore yesterday.. ;-)